samples/zigbee/shell/README.rst

```diff
@@ -0,0 +1,166 @@
+.. _zigbee_shell_sample:
+
+Zigbee: Shell
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+This :ref:`Zigbee <ug_zigbee>` shell sample demonstrates a Zigbee router (with the possibility of being a coordinator) that uses the shell interface inside Zephyr's :ref:`zephyr:shell_api` for interaction.
+
+The sample can be used for several purposes, including:
+
+* Initial configuration of the network - forming a network as a coordinator, adding devices to the network with the install codes, setting the extended PAN ID.
+* Benchmarking - measuring time needed for message to travel from one node to another.
+
+Available CLI interfaces
+************************
+
+The CLI can be accessed by using the following interfaces:
+
+* UART
+* RTT
+
+These interfaces are completely independent one from another and can be used simultaneously or exclusively.
+
+The sample demonstrates the usage of commands listed on :ref:`zigbee_cli_reference`.
+Zigbee CLI is integrated into the system shell that is accessible over the available interfaces.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf5340dk_nrf5340_cpuapp, nrf21540dk_nrf52840
+
+You can use two or more of the development kits listed above and mix different development kits.
```

Comment:
It is unclear to me whether I need an additional DK to set up CLI device as coordinator, or I use the DK I'm going to program with the shell sample?
If I need an additional DK, we need to mention this additional requirement.

Reply:
Can we simplify the testing scenario and just advise a user to use CLI as the coordinator?
There is a separate page (I hope) describing how to replace a Nordic sample -based coordinator with a 3rd-party one.

Reply:
You mean https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_zigbee_other_ecosystems.html ?

Reply:
Fixed!